regimes/pl/extensions.go

```diff
@@ -64,4 +65,45 @@ var extensionKeys = []*tax.KeyDefinition{
 			},
 		},
 	},
+	{
+		Key: ExtKeyKSEFCorrection,
+		Name: i18n.String{
+			i18n.EN: "Zero VAT Extensions for KSeF",
+		},
```

Comment:
So I'm confused with the naming here, I suspect it was most likely copied and pasted incorrectly :-) However, the descriptions below suggest that this extension code has more to do with "date" than configuration of the corrective document. I've not fully understood the meaning. Do you have any more context?

Reply:
Yeah, I forgot to change this description.. This is a field In KSeF which informs the tax authority on when the correction should take legal effect, with values 1-3 meaning:
1 - Changes take effect in the date of the invoice. This is mostly used if you made a mistake in the original invoice.
2 - Changes take effect in the date of the correction. This is used for example when giving a discount or returning items.
3 - Changes take effect in a different date than any of the invoices or the date is different for different elements.